src/app/components/CardInfo/CardInfo.stories.tsx

```diff
@@ -8,7 +8,7 @@ export default {
 
 export const CardSmall = (): JSX.Element => (
   <CardInfo
-    id="3"
+    id={1}
```

Comment:
you don't need curlies if it's just a number

Reply:
didn´t work without the curlies